docs/operator-guide/configuration.md

```diff
@@ -113,6 +113,6 @@ provides links to the relevant documentation for each.
 | Service |
 | ------- |
 | [cloudapi](https://github.com/TritonDataCenter/sdc-cloudapi/blob/master/docs/admin.md#sapi-configuration) |
-| [cnapi](https://github.com/TritonDataCenter/sdc-cnapi/blob/master/docs/index.md#sapi-configuration) |
+| [cnapi](https://github.com/TritonDataCenter/sdc-cnapi/tree/master/docs#sapi-configuration) |
```

Comment:
This should stay `blob` (but `/index.md` does need to be removed.

Reply:
Fixed in 0b2be4d.